<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiderClass</class>
 <widget class="QWidget" name="guiderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Guider</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Drift">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Drift graphics</string>
   </property>
   <widget class="QSpinBox" name="spinBox_XScale">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_YScale">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="l_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X scale(frm.)</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="l_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y scale(&quot;)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Info">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Info</string>
   </property>
   <widget class="QDoubleSpinBox" name="spinBox_GuideRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="l_Focal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxx</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_Aperture">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_FbyD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_FOV">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>YYxYY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Guiding rate,x15&quot;/sec</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Focal,mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aperture,mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>F/D</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FOV,'</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_RecommendedGain">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P=xxxxx</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Control">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QSpinBox" name="spinBox_AccFramesRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_AccFramesDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DirRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>14</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DirDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>14</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DEC</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_PropGainRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_PropGainDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_IntGainRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_IntGainDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_DerGainRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_DerGainDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AverageFrames">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>avg.frm.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartStop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable directions</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Accumulate frames</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proportional gain</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Integral gain</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Derivative gain</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum pulse</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum pulse</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta ,&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse length, ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_DeltaRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_DeltaDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_PulseRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_PulseDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_MaxPulseRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_MaxPulseDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_MinPulseRA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_MinPulseDEC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="l_ErrRA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxx</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sig(RA)&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sig(DEC)&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_ErrDEC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>xxx</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Configuration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QComboBox" name="comboBox_SquareSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Square size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ThresholdAlg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_19">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_XScale</tabstop>
  <tabstop>spinBox_YScale</tabstop>
  <tabstop>comboBox_SquareSize</tabstop>
  <tabstop>comboBox_ThresholdAlg</tabstop>
  <tabstop>spinBox_GuideRate</tabstop>
  <tabstop>checkBox_DirRA</tabstop>
  <tabstop>checkBox_DirDEC</tabstop>
  <tabstop>spinBox_AccFramesRA</tabstop>
  <tabstop>spinBox_AccFramesDEC</tabstop>
  <tabstop>spinBox_PropGainRA</tabstop>
  <tabstop>spinBox_PropGainDEC</tabstop>
  <tabstop>spinBox_IntGainRA</tabstop>
  <tabstop>spinBox_IntGainDEC</tabstop>
  <tabstop>spinBox_DerGainRA</tabstop>
  <tabstop>spinBox_DerGainDEC</tabstop>
  <tabstop>spinBox_MaxPulseRA</tabstop>
  <tabstop>spinBox_MaxPulseDEC</tabstop>
  <tabstop>spinBox_MinPulseRA</tabstop>
  <tabstop>spinBox_MinPulseDEC</tabstop>
  <tabstop>checkBox_AverageFrames</tabstop>
  <tabstop>pushButton_StartStop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
